test/index.spec.ts

```diff
@@ -111,6 +111,7 @@ const lintFixture = async ({
   }
 
   const fixture = ruleFixtures[type];
+  const filePath = `${fixtureFilePath(ruleSet)}/${configuredRule}.${type}.ts`;
```

Comment:
this is unused

Reply:
Removed, thanks!  I was debugging this and left it behind.